index.html

```diff
@@ -23,10 +22,12 @@
        <label for="checkbox_toggle" class="hamburger">&#9776;</label>
        <!-- NAVIGATION MENUS -->
        <div class="menu">
-  <li class="link-li"><a href="#About">About</a></li>
-  <li class="link-li"><a href="#Project">Project</a></li>
-  <li class="link-li"><a href="#Experience">Experience</a></li>
-  <li class="link-li"><a href="#Connect">Connect</a></li>
+  <li class="link-li"><a href="#about">About</a></li>
```

Comment:
There seems to be a z-index issue: I can see the nav menu, but I can't click on it, and the cursor doesn't change when I hover.